stdlib/2and3/plistlib.pyi

```diff
@@ -71,3 +71,5 @@ if sys.version_info >= (3, 8):
         def __index__(self) -> int: ...
         def __reduce__(self) -> Any: ...
         def __hash__(self) -> int: ...
+
+class InvalidFileException(ValueError): ...
```

Comment:
Could you add the `__init__` method, since this does not take an arbitrary amount of arguments like the base class?